cmd/res/configuration.toml

```diff
@@ -10,59 +10,74 @@ Port = 49993
 Protocol = 'http'
 StartupMsg = 'device snmp GO started'
 Timeout = 5000
-ConnectRetries = 10
 Labels = []
 EnableAsyncReadings = true
 AsyncBufferSize = 16
+MaxRequestSize = 0
+UseMessageBus = false
```

Comment:
Need to also add the following to the `[Writable]` section for when running without secure secret store
```
  # InsecureSecrets are required for when Redis is used for message bus
  [Writable.InsecureSecrets]
    [Writable.InsecureSecrets.Sample]
    path = "redisdb"
      [Writable.InsecureSecrets.Sample.Secrets]
      username = ""
      password = ""
```

Reply:
@lenny-intel please help review again, Chris has modified this part with the latest way